fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -445,6 +446,11 @@ private boolean initForwardToLeaderState() {
             }
         }
 
+        // always redirect sql blacklist related queries to the leader
+        if (parsedStmt instanceof AddSqlBlackListStmt || parsedStmt instanceof DelSqlBlackListStmt) {
```

Comment:
Change the function of AddSqlBlackListStmt/DelSqlBlackListStmt like
```
    @Override
    public RedirectStatus getRedirectStatus() {
        return RedirectStatus.FORWARD_NO_SYNC;
    }
```

Only special cases are handled here

Reply:
fixed